src/components/Map/Map.js

```diff
@@ -96,7 +96,7 @@ class OrganizationMap extends Component {
           orgRef.isOpen = false;
         }
 
-        if(Number(i) === index) {
+        if(Number(i) === Number(index)) {
```

Comment:
for what values of `i` & `index` is this need? 
```js
> Number("1") === 1
< true
```

Reply:
would line 95 need this as well?

```js
 if(Number(i) !== index && orgRef.isOpen) 
```

Reply:
Thats a good catch! I actually looked into it and found that both i and index come from the `locationAddressHashTable` I think that we should get rid of the conversions entirely. So it would look like this. 


```
for(var i of orgRef.orgs) {
        if(i !== index && orgRef.isOpen) {
          orgRef.isOpen = false;
        }

        if(i === index) {
          orgRef.isOpen = true
          this.setState({
            center: this.props.organizations[orgRef.orgs[0]].coordinates,
            zoom: 17,
          });
          break;
        }
      }
```